README.md

```diff
@@ -1 +1,8 @@
-# apod-explorer
\ No newline at end of file
+### apod-explorer
+
+`apod-explorer` is an application that serves two primary purposes:
+
+- as a GraphQL-backed API, exposing data access to asynchronously download in bulk high-definition APOD photos
+- as a data visualization UI that enables exploration of APODs using a chosen date range; it'll dynamically download APODs if it's missing the range
```

Comment:
Also, you can further clarify that the API work belongs to the v1 GitHub milestone and the UI front end work belongs to the v2 GitHub milestone.

Reply:
Good idea, done 👍.